### ISSUE TITLE
Advantage over using pandoc/extra?

### ISSUE THREAD

**Issue by JoseTomasTocino** [rstropek/pandoc-latex#15]
First of all, thanks for the effort. I was wondering what's the advantage of using this image instead of using [pandoc/extra](https://hub.docker.com/r/pandoc/extra) which already includes eisvogel and other stuff?

Thanks.

**Comment by rstropek** [comment]
Possibly nothing. Unfortunately, I am not familiar with pandoc/extra. My images has been around for quite a while. There is a good chance that pandoc/extra is a better choice nowadays. Does it work for you when using eisvogel?

**Comment by JoseTomasTocino** [comment]
Yep it works well. Not only does it include eisvogel but also some other useful bits, like pandoc-latex-environments, which allows the usage of, for example, `awesomebox` to create admonition-like boxes and such.
